html/inc/email.inc

```diff
@@ -86,6 +99,36 @@ For further information and assistance with ".PROJECT.", visit
     return send_email($user, $subject, $body);
 }
 
+function send_changed_email($user) {
+    $body_new = "";
+    $body_old = "";
```

Comment:
This is really a minor issue but Scrutinizer is flagging these as major issue. The variables are not used before they are overwritten. You could remove them from here.